### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by katy-mozharovska**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
+
+  it(`should return array [0, 0, 1, 0]
+  if the number of cents equals 10`, () => {
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+  });
+
+  it(`should return array [1, 1, 1, 0]
+  if the number of cents equals 16`, () => {
+    expect(getCoinCombination(16)).toEqual([1, 1, 1, 0]);
+  });
```
Comment (vova-poli) [inline src/getCoinCombination.test.js:39]: would be nice if you added a check for boundary value of 25 [0, 0, 0, 1]

**Review by vova-poli — COMMENTED** [review]
Looks good

File: src/getCoinCombination.test.js (hunk 1 above)
Comment (katy-mozharovska) [reply src/getCoinCombination.test.js:39]: Thanks, added test for 25 cents

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -1,8 +1,50 @@
...
+
+  it(`should return array [1, 0, 0, 0] if the number of cents equals 1`, () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it(`should return array [4, 0, 0, 0] if the number of cents equals 4`, () => {
+    expect(getCoinCombination(4)).toEqual([4, 0, 0, 0]);
+  });
+
+  it(`should return array [5, 0, 0, 0] if the number of cents equals 5`, () => {
```
Comment (Oleksandra-Pasenchenko) [inline src/getCoinCombination.test.js:23]:
```suggestion
  it(`should return array [0, 1, 0, 0] if the number of cents equals 5`, () => {
```

**Review by Oleksandra-Pasenchenko — APPROVED** [review]
Good job, but fix an error in one test description

File: src/getCoinCombination.test.js (hunk 2 above)
Comment (katy-mozharovska) [reply src/getCoinCombination.test.js]: Thanks, fixed
